lib/dpul_collections_web/live/browse_live.ex

```diff
@@ -59,55 +61,26 @@ defmodule DpulCollectionsWeb.BrowseLive do
 
   # If neither, generate a random seed and display random items.
   def handle_params(_params, _uri, socket) do
-    {:noreply, push_patch(socket, to: "/browse?r=#{Enum.random(1..1_000_000)}", replace: true)}
+    r = Enum.random(1..1_000_000)
+    socket = socket |> assign(r: r)
+    {:noreply, push_patch(socket, to: "/browse?r=#{r}")}
```

Comment:
I don't think this is quite right yet. When you go to https://dpul-collections-staging.lib.princeton.edu/, click 'Browse', then click back you still get a new seed and now you can't go back to the home page. Maybe a new `handle_params` that does a replace: true if there's no `r` param, but there's an `r` in the socket, and it uses that `r`? And then still replace: true here.